tests/PhpAmqpLib/QueueTest.php

```diff
@@ -32,4 +33,17 @@
 final class QueueTest extends AbstractQueueTest
 {
     use CreateConnectionTrait;
+
+    /**
+     * @test
+     */
+    public function it_consumes_without_callback()
+    {
+        $this->expectException(QueueException::class);
+
+        $this->exchange->publish('foo');
+        $this->exchange->publish('bar');
+
+        $this->queue->consume(null);
```

Comment:
@prolic check this change
After set timeout in consume method for PhpAmqpLib
```
$this->queue->consume(null);
```
Inside `consume` method - `\PhpAmqpLib\Channel\AbstractChannel::wait` throw AMQPTimeoutException after 2 loop iterations

Reply:
Move this to AbstractQueueTest, if that's a desired behaviour, it should do this for all drivers.

Reply:
> Move this to AbstractQueueTest

Moved from AbstractQueueTest, because after update PhpAmqpLib, behaviour for consume method for null callback changed.

> if that's a desired behaviour

I don't know. For exists callback behavior equal.
Is there a real reason to use null callback?

### Consume behaviour after update PhpAmqpLib
* For null callback
  * AmqpExtension: Doesn't throw any exception
  * PhpAmqpLib: Throw timeout exception
* For exists callback
  * AmqpExtension: Throw timeout exception
  * PhpAmqpLib: Throw timeout exception